data/pokedex.js

```diff
@@ -10217,7 +10217,35 @@ exports.BattlePokedex = {
 		prevo: "spewpa",
 		evoLevel: 12,
 		eggGroups: ["Bug"],
-		otherForms: ["vivillonarchipelago", "vivilloncontinental", "vivillonelegant", "vivillongarden", "vivillonhighplains", "vivillonicysnow", "vivillonjungle", "vivillonmarine", "vivillonmodern", "vivillonmonsoon", "vivillonocean", "vivillonpolar", "vivillonriver", "vivillonsandstorm", "vivillonsavanna", "vivillonsun", "vivillontundra"],
+		otherForms: ["vivillonarchipelago", "vivilloncontinental", "vivillonelegant", "vivillonfancy", "vivillongarden", "vivillonhighplains", "vivillonicysnow", "vivillonjungle", "vivillonmarine", "vivillonmodern", "vivillonmonsoon", "vivillonocean", "vivillonpolar", "vivillonriver", "vivillonsandstorm", "vivillonsavanna", "vivillonsun", "vivillontundra", "vivillonpokeball"],
```

Comment:
Why does Fancy follow alphabetical order but Pokeball is at the end? I'd move Fancy in front of Pokeball.